launch_ros/launch_ros/actions/node.py

```diff
@@ -423,7 +434,7 @@ def execute(self, context: LaunchContext) -> Optional[List[Action]]:
         context.extend_locals({'ros_specific_arguments': ros_specific_arguments})
         ret = super().execute(context)
 
-        if '<node_name_unspecified>' not in self.node_name:
+        if self.is_node_name_fully_specified():
```

Comment:
This if-condition doesn't look equivalent to the condition before this change. Should this be fixed so that it's okay if no namespace was specified?

I think this would change back the original behavior of `test_launch_node_with_name_without_namespace` below.

Reply:
Nevermind. I see what's happening in https://github.com/ros2/launch_ros/pull/153